content/05.use-cases-c.md

```diff
@@ -2,66 +2,58 @@
-As projects or contracts end, there should be a handover period of this code in order for the next cohort of researchers to reuse what was already developed [@doi:10.5334/jors.307; @doi:10.1186/1751-0473-8-7].
+Projects in ecology and evolution often extend beyond the residence of graduate students, research assistants and post-doctoral fellows, who hold limited-term positions [@doi:10.5334/jors.307].
+Without clear plans on project continuitiy, the research code and data management upkeep tends to fall off as researchers move on to new projects or to other institutions.
+Additionally, code and data can be difficult to access when kept only on the personal devices of researchers.
 
-GitHub facilitates project continuity among research code by providing tools that make this handover period easier. 
-As we have already mentioned, using Git for code in Ecology and Evolution can allow for a "paper trail" of sorts to be created for the code, thus allowing for future users of the code access to the entire history of the project [@doi:10.1038/s41559-017-0160]. 
-Additionally, GitHub allows for repositories and organizations to have designated Code Owners [@url:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners]; these code owners can change through time allowing for the transition of code from one cohort of researchers to the next [@doi:10.1890/ES14-00402.1].
-There are also multiple means by which to archive code dependency information with GitHub.
-Dependencies can be described in README files, added to environment.yml and/or an environment.txt file, or stored in a repository using tools such as renv (<https://rstudio.github.io/renv/articles/renv.html>).
-This ensures that as packages continue to develop with new syntax, code will still run as it did when it was developed.
+On one hand, GitHub can facilitate project continuity among research code by making code handover to succeding researchers easier [see @doi:10.5334/jors.307; @doi:10.1186/1751-0473-8-7]. 
+Through its version control, the history of code and data from projects in ecology and evolution can be tracked and be accessible to future laboratory members and collaborators [@doi:10.1038/s41559-017-0160].
+Repositories and organizations can have designated data and code owners [or more appropriate, stewards; see @doi:10.1890/ES14-00402.1] [@url:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners], who can also can change through time allowing for the transition of code from one cohort of researchers to the next.
+
+On the other hand, GitHub allows for informantion and dependencies on software version backwards compatibility to be described and store, to ensure that code will appropriately run as it did when it was developed.
```

Comment:
"GitHub allows for information". 

Is "software version backwards compatibility" correct? It is hard to read - but I guess technically correct.

Reply:
I think that sentence needs to be reworded. I gave a try at revising.